### PR TITLE
centos 8 aware jinja template

### PR DESCRIPTION
**Pull request by Kallio**
packages %end was written no matter what.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,13 +53,20 @@ Optional:
  - kickstart_timezone
  - kickstart_log_host
  - dhcp_kickstart_manage_packages
- - kickstart_packages # if dhcp_kickstart_manage_packages is defined then kickstart_packages must contain %packages
+ - kickstart_packages # if dhcp_kickstart_manage_packages is defined, it must contain at least @Core or @Base
  - kickstart_extra_pre_option
  - kickstart_extra_pre_commands
  - kickstart_extra_post_commands
```
Comment (khappone) [inline README.md:59]: dhcp_kickstart_disable_ipv6: True
could be added to the documentation

File: templates/kickstart.j2 [hunk 2]
```diff
@@ -43,8 +60,9 @@ ignoredisk --only-use={{ hostvars[groups[item][0]]['os_disks'] }}
 {{ line }}
 {% endfor %}
 
-{% if hostvars[groups[item][0]]['dhcp_kickstart_manage_packages'] is not defined %}
 %packages
+{% if [groups[item][0]]['dhcp_kickstart_centos_version'] is "7" %}
```
Comment (khappone) [inline templates/kickstart.j2:64]: should we have "@core" as a default for centos8? Now if I read it correctly, it has to be defined in the group

**Review by khappone — COMMENTED** [review]
Otherwise, looks really good and concise. Have not tested this.

File: README.md [hunk 3]
```diff
@@ -53,13 +53,20 @@ Optional:
  - kickstart_timezone
  - kickstart_log_host
  - dhcp_kickstart_manage_packages
- - kickstart_packages # if dhcp_kickstart_manage_packages is defined then kickstart_packages must contain %packages
+ - kickstart_packages # if dhcp_kickstart_manage_packages is defined, it must contain at least @Core or @Base
```
Comment (junousi) [inline README.md:56]: Not sure why change the first part. This form would imply that `dhcp_kickstart_manage_packages` should contain e.g. `@Core`, whereas the code doesn't look like it.

File: README.md [hunk 4]
```diff
@@ -53,13 +53,20 @@ Optional:
...
  - kickstart_extra_pre_option
  - kickstart_extra_pre_commands
  - kickstart_extra_post_commands
+ - dhcp_kickstart_centos_version set as "8" or "9" in order to skip old centos7 defaults, remember to define kickstart_packages like:
```
Comment (junousi) [inline README.md:60]: FWIW this renders horribly in markdown, at least in Github markdown preview. Matters only if we strive for beauty :) Not requesting a change.

File: README.md (hunk 1 above)
Comment (junousi) [reply README.md:59]: But it is already there (though with "set as False if needed")

File: templates/kickstart.j2 [hunk 5]
```diff
@@ -1,6 +1,11 @@
 install
 text
+reboot
+skipx
+
+{% if hostvars[groups[item][0]]['dhcp_kickstart_disable_ipv6'] is True %}
 network --noipv6
```
Comment (junousi) [inline templates/kickstart.j2:7]: If this is made conditional, shouldn't there be two cases

`network`

and

`network --noipv6`

?

Or is it default to run `network` if it's missing?

File: templates/kickstart.j2 [hunk 6]
```diff
@@ -22,15 +27,27 @@ repo --name={{ repo.name }} --baseurl={{ repo.url }}
...
 selinux --disabled
-reboot
 firewall --service=ssh
-skipx
 services --enabled=ntpd
+{% endif %}
+
+{% if [groups[item][0]]['dhcp_kickstart_centos_version'] is "8" %}
+selinux --permissive
+firewall --enabled --ssh
```
Comment (junousi) [inline templates/kickstart.j2:49]: I trust you know what you're doing by not requiring chrony here.

File: templates/kickstart.j2 (hunk 2 above)
Comment (junousi) [reply templates/kickstart.j2:64]: This would be a good addition.

But I'm ok with this as long as it's documented in the Readme.

**Review by junousi — COMMENTED** [review]
Added some comments/questions, mostly cosmetic.

My only real concern is the network vs. network --noipv6 part.

**Review by junousi — APPROVED** [review]
LGTM

**Pull request merged**